python/tests/db/test_db.py

```diff
@@ -42,22 +43,23 @@ def test_getitem(self):
 
     def test_get_default_database(self):
         assert self.c1.get_default_database().name == 'my_database'
-        with pytest.raises(pymongo.errors.ConfigurationError, match = 'No default database'):
+        with pytest.raises(pymongo.errors.ConfigurationError, match='No default database'):
             self.c2.get_default_database()
 
     def test_get_database(self):
         assert self.c1.get_database().name == 'my_database'
         assert self.c2.get_database('my_db').name == 'my_db'
-        with pytest.raises(pymongo.errors.ConfigurationError, match = 'No default database'):
+        with pytest.raises(pymongo.errors.ConfigurationError, match='No default database'):
             self.c2.get_database()
 
+
 class TestDatabase():
 
     def setup_class(self):
         enable_gridfs_integration()
         client = mongomock.MongoClient('localhost')
```

Comment:
Maybe we should just use the new MongoDB installed into the CI instead of mongomock? We should probably remove mongomock from our CI setup too.

Reply:
done